chart/values.yaml

```diff
@@ -2707,8 +2707,9 @@ config:
     statsd_port: 9125
     statsd_prefix: airflow
     statsd_host: '{{ printf "%s-statsd" (include "airflow.fullname" .) }}'
-  webserver:
+  fab:
     enable_proxy_fix: 'True'
+  webserver:
     # For Airflow 1.10
     rbac: 'True'
```

Comment:
Is this backward compatible with older airflow versions?
the chart support airflow 1.0+

Reply:
Good point, then no. I'll update it

Reply:
Fixed, I duplicated the config in `fab` and `webserver`